Library/RadarIO.Xamarin.Lib/RadarSDK.cs

```diff
@@ -129,13 +128,15 @@ public class RadarAddress
         public string Borough;
         public string County;
         public string Neighborhood;
-#if MONOANDROID
-        public string Street;
-#endif
         public string Number;
         public string AddressLabel;
         public string PlaceLabel;
         public RadarAddressConfidence Confidence;
+
+        /// <summary>
+        /// Android-only
```

Comment:
This isn't Android only.  Is there a missing iOS caller?

Reply:
Interesting, yeah, it looks like that field might be missing from iOS.  Looks like an oversight.